ludwig/backend/_ray210_compat.py

```diff
@@ -143,36 +140,3 @@ def _expected_utilization(self, cpus_per_trial, cpus_total):
             )
         )
         return (actual_concurrency * cpus_per_trial) / (cpus_total + 0.001)
-
-
-class HorovodTrainerRay210(HorovodTrainer):
-    """HACK(geoffrey): This is a temporary fix to support Ray 2.1.0.
-
-    Specifically, this Trainer ensures that TunerRay210 is called by the class.
-    For more details, see TunerRay210.
-    """
-
-    def fit(self) -> Result:
-        """Runs training.
-
-        Returns:
-            A Result object containing the training result.
-
-        Raises:
-            TrainingFailedError: If any failures during the execution of
-            ``self.as_trainable()``.
-        """
-        from ray.tune.error import TuneError
-
-        trainable = self.as_trainable()
-
-        tuner = TunerRay210(trainable=trainable, run_config=self.run_config)
-        result_grid = tuner.fit()
-        assert len(result_grid) == 1
-        try:
-            result = result_grid[0]
-            if result.error:
-                raise result.error
-        except TuneError as e:
-            raise TrainingFailedError from e
-        return result
```

Comment:
Do we officially want to drop support for Ray 2.1?

Reply:
Oh I see, just moved it a different place, all good

Reply:
Maybe when 2.4 comes out. It would also need to be in v0.8, not 0.7.1.